nuklear_console_spacing.h

```diff
@@ -0,0 +1,61 @@
+#ifndef NK_CONSOLE_SPACING_H__
+#define NK_CONSOLE_SPACING_H__
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+NK_API nk_console* nk_console_spacing(nk_console* parent, int cols);
+NK_API struct nk_rect nk_console_spacing_render(nk_console* widget);
+
+#if defined(__cplusplus)
+}
+#endif
+
+#endif  // NK_CONSOLE_SPACING_H__
+
+#if defined(NK_CONSOLE_IMPLEMENTATION) && !defined(NK_CONSOLE_HEADER_ONLY)
+#ifndef NK_CONSOLE_SPACING_IMPLEMENTATION_ONCE
+#define NK_CONSOLE_SPACING_IMPLEMENTATION_ONCE
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+NK_API struct nk_rect nk_console_spacing_render(nk_console* widget) {
+    nk_console_layout_widget(widget);
+
+    struct nk_rect widget_bounds = nk_layout_widget_bounds(widget->ctx);
```

Comment:
We can tackle these as follow ups... 

Since spacing doesn't leverage the mouse at all, I think we can skip getting it's bounds.